src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FIgureSupplier figureSupplier = new FIgureSupplier();
```

Comment:
The class name 'FIgureSupplier' should start with a lowercase letter following the camelCase convention. Also, there is a typo in the class name; it should be 'FigureSupplier'.